src/plopp/plotting/common.py

```diff
@@ -192,7 +214,12 @@ def preprocess(
     coords:
         If supplied, use these coords instead of the input's dimension coordinates.
     """
-    out = to_data_array(obj)
+    if isinstance(coords, str):
+        coords = [coords]
```

Comment:
I think this means that the check
```Py
        if isinstance(coords, str):
            coords = [coords]
```
further down (L231) can be removed?

Reply:
Yes, removed